suite-common/message-system/config/config.v1.json

```diff
@@ -814,6 +814,63 @@
                     }
                 }
             }
+        },
+        {
+            "conditions": [
```

Comment:
I would do this in separate PR, it is kinda dangerous as we can forget to bump version/release config,...

Reply:
Since we are waiting for FW... let's wait and see.